### PR TITLE
Support LLM Guard content mod across Anthropic/Gemini/Bedrock/Azure/Groq/etc. (via LiteLLM)

### PR DESCRIPTION
**Pull request by krrishdholakia**
## Change Description

add litellm tutorial to docs. LiteLLM Proxy is an OpenAI-compatible proxy, which works across 100+ LLMs.  The tutorial shows how to use LLM Guard to moderate calls across those LLMs. 

https://github.com/BerriAI/litellm

## Issue reference

n/a

## Checklist

- [x] I have reviewed the [contribution guidelines](../CONTRIBUTING.md)
- [x] My PR contains documentation updates / additions if required

### CODE REVIEW

**Comment by krrishdholakia** [comment]
cc: @asofter @CandiedCode

**Review by asofter — APPROVED** [review]
Thank you for the contribution @krrishdholakia ! And also for the integration itself.

**Comment by krrishdholakia** [comment]
@asofter working on it

File: docs/tutorials/litellm.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+# Apply LLM Guard Content Mod across 100+ LLMs w/ LiteLLM
+
+Use LLM Guard with LiteLLM Proxy to moderate calls across Anthropic/Bedrock/Gemini/etc. LLMs with [LiteLLM](https://github.com/BerriAI/litellm)
```
Comment (CandiedCode) [inline docs/tutorials/litellm.md:3]:
```suggestion
Use LLM Guard with LiteLLM Proxy to moderate calls across Anthropic/Bedrock/Gemini/etc. LLMs with [LiteLLM](https://github.com/BerriAI/litellm).
```

**Review by asofter — APPROVED** [review]

File: docs/tutorials/litellm.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+
+LiteLLM currently supports requests in:
+- [The OpenAI format](https://docs.litellm.ai/docs/completion/input) - `/chat/completion`, `/embedding`, `completion`, `/audio/transcription`, etc.
+- [The Anthropic format](https://docs.litellm.ai/docs/anthropic_completion) - `/messages`
+
+## Quick Start
+
+Let's add LLM Guard content mod for Anthropic API calls
```
Comment (CandiedCode) [inline docs/tutorials/litellm.md:12]:
```suggestion
Let's add LLM Guard content mod for Anthropic API calls.
```

File: docs/tutorials/litellm.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+Set the LLM Guard API Base in your environment
```
Comment (CandiedCode) [inline docs/tutorials/litellm.md:14]:
```suggestion
Set the LLM Guard API Base in your environment.
```

File: docs/tutorials/litellm.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+```bash
+export LLM_GUARD_API_BASE="http://0.0.0.0:8192" # deployed llm guard api
+export ANTHROPIC_API_KEY="sk-..." # anthropic api key
+```
+
+Add `llmguard_moderations` as a callback
```
Comment (CandiedCode) [inline docs/tutorials/litellm.md:21]:
```suggestion
Add `llmguard_moderations` as a callback.
```

File: docs/tutorials/litellm.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    litellm_params: # all params accepted by litellm.completion() - https://docs.litellm.ai/docs/completion/input
+      model: anthropic/claude-3-5-sonnet-20240620 ### MODEL NAME sent to `litellm.completion()` ###
+      api_key: os.environ/ANTHROPIC_API_KEY
+
+
+litellm_settings:
+    callbacks: ["llmguard_moderations"]
+```
+
+Now you can easily test it
```
Comment (CandiedCode) [inline docs/tutorials/litellm.md:35]:
```suggestion
Now you can easily test it:
```

**Pull request merged**